packages/react/src/BlockSideMenu/components/BlockSideMenu.tsx

```diff
@@ -69,24 +71,24 @@ export const BlockSideMenu = (props: BlockSideMenuProps) => {
             </ActionIcon>
           </div>
         </Menu.Target>
-        <Menu.Dropdown className={classes.root}>
-          <Menu.Item
-            component={"div"}
-            onClick={() => {
-              setDragHandleMenuOpened(false);
-              props.unfreezeMenu();
-              props.deleteBlock();
-            }}>
-            Delete
-          </Menu.Item>
-          <ColorPickerMenu
-            onClick={() => {
+        {CustomDragHandleMenu ? (
+          <CustomDragHandleMenu
+            editor={props.editor}
+            closeMenu={() => {
               setDragHandleMenuOpened(false);
               props.unfreezeMenu();
             }}
-            {...props}
           />
-        </Menu.Dropdown>
+        ) : (
+          <DragHandleMenu>
```

Comment:
Let's make this a little cleaner by separating this to a separate file, and then use
`DragHandleMenu = props.dragHandleMenu || DefaultDragHandleMenu`.